src/components/Navbar/SettingsDrawer/AudioSection.tsx

```diff
@@ -78,11 +86,24 @@ const AudioSection = () => {
                 orientation={RadioGroupOrientation.Horizontal}
               />
             </Section.Row>
+            <Section.Row>
+              <Section.Label>Playback Speed</Section.Label>
+              <Select
+                id="theme-section"
+                name="theme"
+                options={playbackRates}
+                value={playbackRate}
+                onChange={setPlaybackRate}
+              />
+            </Section.Row>
           </Section>
         )}
       />
     </div>
   );
 };
 
+const DEFAULT_PLAYBACK_RATE = '1';
+const playbackRates = generateSelectOptions(['0.5', '1', '1.5', '2']);
```

Comment:
Nice! Can we use these values? Taken from youtube.

Can we also display "Normal" when the value is 1? I think that it'll be clearer for less technical users

![image](https://user-images.githubusercontent.com/11590314/136687670-d828c2e4-dcd2-4c3c-af12-c314ea2e603e.png)